js/project.js

```diff
@@ -0,0 +1,34 @@
+alert("hello world");
+
+$("nav a").click(function(){
+	$("nav .current").removeClass("current");
+	$(this).addClass("current");
+});
+
+//$("#submit").click(function(){
+//$(".ipsum").slideToggle(400);
+//$("p").slideToggle(400);
+//});
+
+$("#submit").click(function(){
+	var ipsumType = $('nav a.current').attr('data-ipsum');
+	$('#' + ipsumType).slideDown();
+	var numParagraphs = $('#paragraphs').val();
+	console.log(numParagraphs);
```

Comment:
@sabgab almost there. See if you can finish off the missing pieces starting at line 17.